articles/liquity/Asset Risk Assessment - bLUSD.md

```diff
@@ -0,0 +1,203 @@
+
+## Exiting the bonds - Chicken_out
+
+When someone chicken_out the LUSD is paid back by withdrawing the investment positions in the B.Protocol. At the time of heavy liquidation, some Chicken Outs might temporarily revert if the user's bond is greater than the LUSD available.
+
+![Chicken out may revert due to heavy liquidations](https://lh3.googleusercontent.com/PPOAewlKcG5gifPmT6NGino5Bjcyoh4mot-L5Uqcb3EADn84KgfgQ4u3c-P4RXgbQ06izB0bNpRS8sl3FOki4K05Y8tEKWkCRV0cQt-Eu-_jFfKysR8kgX0JgOK-_F4ozjURNH20dopGluUsLm6_WS5YCDg4FUqRo_pMvyrgoSqLkf4XItIiRLrMGukqIg)
+
+# Risks to LPs of bLUSD-LUSD3CRV
+
+The scope was limited to LUSD and bLUSD as a product for this risk assessment. Here are the liquidity pools in consideration: [bLUSD-LUSD3CRV](https://curve.fi/factory-crypto/134) (v2 pool), [LUSD3CRV](https://curve.fi/lusd) (v1 pool), & [3POOL](https://curve.fi/3pool) (v1 pool).
+
+Chicken bonds are in a sense transferring the yield from the bonded funds to the previous bondholders having bLUSD a.k.a the amplified yields.
+
+Other than smart contract risk, LP might face a risk associated with bLUSD price fluctuation, LUSD peg, risks at the point of yield generation, and other assets in the liquidity pool.
+
+### Impermanent Loss
+
+Impermanent loss is basically the loss that is equivalent to the upside gained on holding the token instead of providing liquidity. In other words, it is an opportunity cost that may occur while providing liquidity, due to the price shift of one asset of the liquidity pool, versus holding the assets. It does depend on the ratio in which the liquidity was provided, the amount of liquidity, and the depth of that liquidity pool.
+
+Ideally, bLUSD’s ever-increasing floor price should drive the market price up and LUSD being a pegged asset there exists a clear impermanent loss while providing liquidity.
+
+While providing liquidity as 50% in bLUSD and 50% in LUSD, if the market price of bLUSD increases, the ROI generated without considering the trading fees would be less than simply holding bLUSD for the same value.
+
+### The risk associated with the bLUSD price
+
+As we discussed in the earlier section, due to the factors like the creation of fewer bonds, lower yield generation and accelerated bLUSD accrual there exists a chance that the market price of bLUSD drops.
+
+### The risk associated with the LUSD price
+
+When people are not confident in holding ETH in a flash crash there exists a chance that LUSD might get depegged. Based on the historic performance of LUSD, this sounds highly unlikely but it can’t be ignored.
+
+### The risk associated with the yield-generating protocols
+
+bLUSD is exposed to the risks at B.Protocol and Yearn used for generating yield. Loss of funds from these protocol/vaults for any reason can severely impact the bLUSD floor price which is the biggest demand driver for holding blUSD or bonding LUSD.
```

Comment:
I think there could be a bit more here on the risk from the associated protocols.
Is B.protocol audited? How long have they been around?
How long has the yearn strategy they rely on been around?